src/components/region.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Card from 'react-bootstrap/Card';
+import { Link } from 'react-router-dom';
+
+const Region = ({ region }) => {
+  const {
+    name,
+  } = region;
+  const totalConfirmed = region.today_confirmed;
+  return (
+    <Card key={name} className="col-sm-12 col-lg-6 col-xl-3 text-center">
```

Comment:
- [ ] I would recommend you follow the design as provided. In the original design, on the mobile screen width, the countries section is arranged to have two countries in one row, but your implementation only has one per row, kindly fix that. Here, I think having `col-6` instead of `col-sm-12` will fix that.

- [ ] Also, kindly implement the alternating color thickness as highlighted in the original design. **NOTE:** I am NOT asking you to change your color.
- [ ] Kindly ensure these changes also reflect on the live demo as well.

  Original design | Your Design
  :---:|:---:
  ![jonathas-orig-home](https://user-images.githubusercontent.com/36542986/152301395-3b7584b7-0994-4941-9c32-e4a480394f90.jpg) | ![jonathas-own-home](https://user-images.githubusercontent.com/36542986/152300594-a54838ad-92fa-4a82-98b7-e498f0591318.jpg)